addons/website_profile/static/src/components/profile_dialog/profile_dialog.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="website_profile.ProfileDialog" t-inherit="web.ConfirmationDialog" t-inherit-mode="primary">
+        <xpath expr="//Dialog" position="attributes">
+            <attribute name="size">xl</attribute>
+        </xpath>
+        <xpath expr="//p[@t-out='props.body']" position="replace">
+            <input t-ref="upload" type="file" class="d-none" name="ufile" accept="image/*" t-on-change="onChangeProfileImg"/>
+            <div class="row">
+                <div class="col-12 col-sm-3 mb-3">
+                    <div class="card o_card_people">
+                        <div class="card-body">
+                            <img class="o_wforum_avatar_img w-100 mb-3" t-ref="profileImg" t-attf-src="/web/image/res.users/#{user.id}/avatar_128"/>
+                            <div class="text-center">
+                                <a href="#" title="Edit" aria-label="Edit" class="btn btn-primary" t-on-click="onEditProfileImg">
+                                    <i class="fa fa-pencil fa-1g float-sm-none float-md-start"/>
+                                </a>
+                                <a href="#" title="Clear" aria-label="Clear" class="btn border-primary" t-on-click="onClearProfileImg">
+                                    <i class="fa fa-trash-o float-sm-none float-md-end"></i>
+                                </a>
```

Comment:
![image](https://github.com/user-attachments/assets/785e3c01-1427-43fd-a674-5ca2d7a0958a)

Maybe a bit of spacing?